### ISSUE TITLE
Date, Month, Year choose the previous one

### ISSUE THREAD

**Issue by alexkids24** [cletourneau/angular-bootstrap-datepicker#3]
When you choose a day, a month, a year it returns always the previous one, this is just because it returns UTCDay, UTCMonth, when the date is not created in non UTC.

**Comment by veerendrakumar-github** [comment]
can anybody solve this issue,i am trying to resolve but not able to fix this issue
anyone get idea let me know please,thank you.

**Comment by veerendrakumar-github** [comment]
yes i solved my self,problem is plugin code.I changed plugin now working fine without any issues

**Comment by mahlatsem** [comment]
I only noticed an issue with day for me, debugging let me to the following line:

this._trigger('changeDate');

I got rid of it, all seems fine now, just hoping i didn't overlook something else

**Comment by mahlatsem** [comment]
Got around this by changing "d: date.getUTCDate()" to "d: date.getDate()"

**Comment by flyher** [comment]
@mahlatsem 
still does not work~~~

@veerendrakumar-github
do you have any idea?please share here,3ks.

**Comment by flyher** [comment]
@alexkids24 
hi
#6 
has solved this bug.
